BasicSample/Assets/Shared/Scripts/MixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.OpenXR.Remoting;
+using System;
+using System.IO;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.XR.Management;
+using UnityEditor.XR.OpenXR.Features;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    enum MixedRealityProjectConfiguration
+    {
+        None,
+        RunNativelyonPCVR,
+        RunNativelyonHL2,
+        RunRemotelyonUWP,
+        RunRemotelyonWin32
+    }
+    public class MixedRealityProjectSelectionWindow : EditorWindow
+    {
+        private MixedRealityProjectConfiguration m_selectedMRConfiguration;
+        private bool m_disablePopup;
+        private const float Default_Window_Height = 300.0f;
+        private const float Default_Window_Width = 400.0f;
+
+        public static MixedRealityProjectSelectionWindow Instance { get; private set; }
+        public static bool IsOpen => Instance != null;
+        private static bool ShowfromMenu = false;
+        private static PopupUserSettings UserSettings;
+        private const string SettingsFileName = "MixedRealityOpenXRProjectSelectionSettings.asset";
+        private static string UserSettingsFolder => Path.Combine(Application.dataPath, "..", "UserSettings");
+        private static string SettingsAssetPath => Path.Combine(UserSettingsFolder, SettingsFileName);
+
+        [MenuItem("Mixed Reality/Quick Setup", false, 499)]
+        private static void ShowWindowFromMenu()
+        {
+            ShowfromMenu = true;
+            ShowWindow();
+        }
+
+        public static void ShowWindow()
+        {
+            GetUserSettings();
+            // There should be only one configurator window open as a "pop-up". If already open, then just force focus on our instance
+            if (IsOpen)
+            {
+                Instance.Focus();
+            }
+            else
+            {
+                if(!UserSettings.DisablePopup || ShowfromMenu)
+                {
+                    var window = CreateInstance<MixedRealityProjectSelectionWindow>();
+                    window.titleContent = new GUIContent("MixedReality Project Selection Window", EditorGUIUtility.IconContent("_Popup").image);
```

Comment:
I really like the "Quick Setup" name - maybe we could use it here too?
It could also be good to clarify this is a script specific to the samples.

```suggestion
                    window.titleContent = new GUIContent("MixedReality Sample Quick Setup", EditorGUIUtility.IconContent("_Popup").image);
```

Reply:
Another option could be just having the window be simply titled "Quick Setup" and let the text in the window speak for itself

Reply:
haha, I initially had it named "Quick Setup" but Oliver suggested otherwise.

Reply:
Did I 😄 ?  sorry about that.  I'm warming up to this name now.  Sounds good.

Reply:
addressed

Reply:
addressed